testdata/p4_16_samples_outputs/issue2485-midend.p4

```diff
@@ -0,0 +1,34 @@
+#include <core.p4>
+
+header H {
+    bit<16> a;
+}
+
+struct metadata {
+}
+
+struct Headers {
+    H h;
+}
+
+parser p(packet_in packet, out Headers hdr) {
+    state start {
+        packet.extract<H>(hdr.h);
+        transition select(hdr.h.a) {
+            16w0x0 &&& 16w0xffff: parse;
+            16w0x1 &&& 16w0xffff: parse;
+            16w0x2 &&& 16w0xfffe: parse;
+            16w0x4 &&& 16w0xffff: parse;
```

Comment:
This range can be represented using only 2 ternary value/masks, i.e. (16w0x0 &&& 16w0xfffc) and (16w0x4 &&& 16w0xffff).  It seems odd to represent it using more ternary value/masks than needed.

Reply:
This is an issue when the start of Range is zero.  I will have to mull over this one.  Right now, the exhaustive test you wrote in range-to-tcam-entries.c is fine with four entries.